core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiService.scala

```diff
@@ -79,13 +79,6 @@ trait WorkspaceApiService extends UserInfoDirectives {
           }
         } ~
         path("workspaces" / Segment / Segment) { (workspaceNamespace, workspaceName) =>
-          /* we enforce a 6-character minimum for workspaceNamespace, as part of billing project creation.
```

Comment:
This comment hasn't been true since the paths were re-ordered for correct path specificity.
In this PR however, we can finally demonstrate that this is true.